### DISCUSSION TITLE
Output of batch processing has random filenames

### DISCUSSION THREAD

**Discussion by Zhincore** [glucauze/sd-webui-faceswaplab#20] [Q&A] [answered]
Output of batch processing has random filenames, making it more difficult to match the pictures to the source ones. 
Did I set something wrong or is this the current behavior?

**Comment by glucauze** [comment, marked as answer]
No, you're right, I generate a new name for each file. I'll see if I can fix that in a future version. It's going to be a pain to modify because of the logic used to manage the batch both in the interface and in the API in 1.2.0. I'll see about that later.
